app/scripts/metamask-controller.js

```diff
@@ -2351,11 +2373,10 @@ export default class MetamaskController extends EventEmitter {
         seedPhraseAsBuffer,
       );
 
-      const ethQuery = new EthQuery(this.provider);
       accounts = await keyringController.getAccounts();
       lastBalance = await this.getBalance(
```

Comment:
Couldn't `getBalance` just access `this.ethQuery` internally?

Reply:
ping on this question again

Reply:
oh, no, the code changes are still there, but they can be removed

Reply:
removed in 71d9f7ad52